setup.py

```diff
@@ -1,93 +1,38 @@
 #!/usr/bin/env python
 
-from __future__ import with_statement
-
-import os
-try:
-    from ConfigParser import ConfigParser, DuplicateSectionError  # Python 2
-except ImportError:
-    from configparser import ConfigParser, DuplicateSectionError  # Python 3
-
 from glob import glob
 from collections import defaultdict
-
-# io.StringIO is strictly unicode only. Python 2 StringIO.StringIO accepts
-# bytes, so we'll conveniently ignore decoding and reencoding the file there.
-try:
-    from StringIO import StringIO  # Python 2
-except ImportError:
-    from io import StringIO  # Python 3
-
-# Graphite historically has an install prefix set in setup.cfg. Being in a
-# configuration file, it's not easy to override it or unset it (for installing
-# graphite in a virtualenv for instance).
-# The prefix is now set by ``setup.py`` and *unset* if an environment variable
-# named ``GRAPHITE_NO_PREFIX`` is present.
-# While ``setup.cfg`` doesn't contain the prefix anymore, the *unset* step is
-# required for installations from a source tarball because running
-# ``python setup.py sdist`` will re-add the prefix to the tarball's
-# ``setup.cfg``.
-with open('setup.cfg', 'r') as f:
-    orig_setup_cfg = f.read()
-cf = ConfigParser()
-cf.readfp(StringIO(orig_setup_cfg), 'setup.cfg')
-
-if os.environ.get('GRAPHITE_NO_PREFIX') or os.environ.get('READTHEDOCS'):
-    cf.remove_section('install')
-else:
-    try:
-        cf.add_section('install')
-    except DuplicateSectionError:
-        pass
-    if not cf.has_option('install', 'prefix'):
-        cf.set('install', 'prefix', '/opt/graphite')
-    if not cf.has_option('install', 'install-lib'):
-        cf.set('install', 'install-lib', '%(prefix)s/webapp')
-
-with open('setup.cfg', 'w') as f:
-    cf.write(f)
-
-if os.environ.get('USE_SETUPTOOLS'):
-  from setuptools import setup
-  setup_kwargs = dict(zip_safe=0)
-
-else:
-  from distutils.core import setup
-  setup_kwargs = dict()
+from setuptools import setup
+import os
 
 
 storage_dirs = []
 
-for subdir in ('whisper/dummy.txt', 'ceres/dummy.txt', 'rrd/dummy.txt', 'log/dummy.txt', 'log/webapp/dummy.txt'):
-  storage_dirs.append( ('storage/%s' % subdir, []) )
-
-webapp_content = defaultdict(list)
+for subdir in ('whisper/dummy.txt', 'ceres/dummy.txt', 'rrd/dummy.txt',
+               'log/dummy.txt', 'log/webapp/dummy.txt'):
+    storage_dirs.append(('storage/%s' % subdir, []))
```

Comment:
since you're editing this, might as well clean it up a bit:

```python
for subdir in ('whisper', 'ceres', 'rrd', 'log', 'log/webapp'):
    storage_dirs.append(('storage/%s/dummy.txt' % subdir, []))
```

also not sure if bot "log" and "log/webapp" are needed

Reply:
it also might be nice to use ".keep" instead of "dummy.txt"

Reply:
cleaned up. Not sure what happens wit the .keep vs dummy.txt when updating. Will leave it like this for now.